test/e2e/persistent_volumes.go

```diff
@@ -749,6 +853,33 @@ func verifyGCEDiskAttached(diskName string, nodeName types.NodeName) bool {
 	return isAttached
 }
 
+// Sanity check for vSphere testing.  Verify the persistent disk attached to the node.
+func verifyVsphereDiskAttached(volumePath string, nodeName types.NodeName) bool {
```

Comment:
These functions are already available in vsphere_Volume_util.go. Use those functions.